common/src/com/thoughtworks/go/config/materials/tfs/TfsMaterial.java

```diff
@@ -88,6 +88,10 @@ public MaterialConfig config() {
         return new TfsMaterialConfig(url, userName, domain, getPassword(), projectPath, goCipher, autoUpdate, filter, folder, name);
     }
 
+    public String getDomain() {
```

Comment:
i think this getter is useful. every materials every field is exposed with a getter apart from TFS - domain & project-path.